userspace/falco/stats_writer.cpp

```diff
@@ -336,7 +336,9 @@ void stats_writer::collector::get_metrics_output_fields_wrapper(
 	if (machine_info)
 	{
 		output_fields["evt.hostname"] = machine_info->hostname; /* Explicitly add hostname to log msg in case hostname rule output field is disabled. */
-		output_fields["falco.host_boot_ts"] = machine_info->boot_ts_epoch;
+		// This line generates a SIGTRAP in zig debug builds if the casting is removed.
+		// It seems caused by the pragma pack for the scap_machine_info structure.
+		output_fields["falco.host_boot_ts"] = (uint64_t)machine_info->boot_ts_epoch;
```

Comment:
As weird as it seems, this stupid cast fixed a SIGTRAP generated on debug builds by zig.
If i removed the `pragma pack` from the scap_machine_info struct, the issue went away. Most probably this is just a bug somewhere in how zig/clang compile the sources.